src/TestSuite/Stub/ConsoleOutput.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * CakePHP(tm) Tests <http://book.cakephp.org/2.0/en/development/testing.html>
+ * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @since         3.0.0
+ * @license       http://www.opensource.org/licenses/mit-license.php MIT License
+ */
+
+namespace Cake\TestSuite\Stub;
+
+use Cake\Console\ConsoleOutput as Base;
+
+/**
+ * A ConsoleOutput class intended for test cases.
+ */
+class ConsoleOutput extends Base
```

Comment:
@markstory I've modified the 3.1 version to simulate $newilnes > 1 (ignoring newlines = 0) and to account for usage with an array.